src/data-workspace/category-combo-table-body/category-combo-table-body.test.js

```diff
@@ -0,0 +1,770 @@
+        zeroIsSignificant: false,
+    },
+    FTRrcoaog83: {
+        aggregationType: 'SUM',
+        categoryCombo: { id: 'bjDvmb4bfuf' },
+        code: 'DE_1148614',
+        created: '2011-04-20T17:54:58.775',
+        displayFormName: 'Accute Flaccid Paralysis (Deaths < 5 yrs)',
+        displayName: 'Accute Flaccid Paralysis (Deaths < 5 yrs)',
+        displayShortName: 'Accute Flaccid Paral (Deaths < 5 yrs)',
+        id: 'FTRrcoaog83',
+        lastUpdated: '2017-05-19T15:13:52.488',
+        name: 'Accute Flaccid Paralysis (Deaths < 5 yrs)',
+        valueType: 'NUMBER',
+    },
+}
+
+const categoryCombos = {
+    dzjKKQq0cSO: {
+        id: 'dzjKKQq0cSO',
+        categories: ['fMZEcRHuamy', 'YNZyaJHiHYq'],
+        categoryOptionCombos: [
+            {
+                categoryOptions: ['wbrDrL2aYEc', 'btOyqprQ9e8'],
+                code: 'COC_290',
+                displayName: 'Outreach, <1y',
+                id: 'V6L425pT3A0',
+                name: 'Outreach, <1y',
+            },
+            {
+                categoryOptions: ['wbrDrL2aYEc', 'GEqzEKCHoGA'],
+                code: 'COC_289',
+                displayName: 'Outreach, >1y',
+                id: 'hEFKSsPV5et',
+                name: 'Outreach, >1y',
+            },
+            {
+                categoryOptions: ['qkPbeWaFsnU', 'btOyqprQ9e8'],
+                code: 'COC_292',
+                displayName: 'Fixed, <1y',
+                id: 'Prlt0C1RF0s',
+                name: 'Fixed, <1y',
+            },
+            {
+                categoryOptions: ['qkPbeWaFsnU', 'GEqzEKCHoGA'],
+                code: 'COC_291',
+                displayName: 'Fixed, >1y',
+                id: 'psbwp3CQEhs',
+                name: 'Fixed, >1y',
+            },
+        ],
+        created: '2011-12-24T12:24:25.203',
+        dataDimensionType: 'DISAGGREGATION',
+        displayName: 'Location and age group',
+        favorite: false,
+        isDefault: false,
+        lastUpdated: '2011-12-24T12:24:25.203',
+        name: 'Location and age group',
+    },
+    bjDvmb4bfuf: {
+        categories: ['GLevLNI9wkl'],
+        categoryOptionCombos: [
+            {
+                code: 'default',
+                name: 'default',
+                categoryOptions: ['xYerKDKCefk'],
+                displayName: 'default',
+                id: 'HllvX50cXC0',
+            },
+        ],
+        code: 'default',
+        created: '2011-12-24T12:24:25.203',
+        dataDimensionType: 'DISAGGREGATION',
+        displayName: 'default',
+        favorite: false,
+        id: 'bjDvmb4bfuf',
+        isDefault: true,
+        lastUpdated: '2016-04-12T20:37:49.126',
+        name: 'default',
+    },
+}
+
+const metadata = {
+    categories,
+    categoryOptions,
+    categoryCombos,
+    dataElements,
+}
+
+describe('<CategoryComboTableBody />', () => {
+    useMetadata.mockReturnValue({ data: metadata })
+
+    it('should render 15 input rows and 5 columns per input', () => {
```

Comment:
Yeah, I like it when the description explains exactly what I'm expecting. But I suppose you can also get that information when looking at the `expect` calls. I've copied all suggestions (incl. the ones below)

Reply:
Nice, thanks. I think it's nice that the test describe what the component/what you're testing is supposed to do. As it can give an overview of the functionality. Also what I've seen most libraries do.